core/src/main/java/it/pagopa/selfcare/mscore/core/strategy/factory/OnboardingInstitutionStrategyFactory.java

```diff
@@ -217,7 +217,10 @@ private Consumer<OnboardingInstitutionStrategyInput> sendEmailWithoutDigestOrRol
         return strategyInput -> {
             try {
                 User user = userService.retrieveUserFromUserRegistry(strategyInput.getPrincipal().getId(), EnumSet.allOf(User.Fields.class));
-                notificationService.sendMailForApprove(user, strategyInput.getOnboardingRequest(), strategyInput.getOnboardingRollback().getToken().getId());
+                    notificationService.sendMailForApprove(user, strategyInput.getOnboardingRequest(), strategyInput.getOnboardingRollback().getToken().getId());
+                    if(InstitutionType.PT.equals(strategyInput.getOnboardingRequest().getInstitutionUpdate().getInstitutionType())){
+                        notificationService.sendMailToPT(user, strategyInput.getInstitution(), strategyInput.getOnboardingRequest());
```

Comment:
you might avoid to reference to "PT" for the method, it send a registration mail so you could name for example **sendRegistrationMail**